config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/IdentityProviderEntityBuilder.java

```diff
@@ -168,6 +178,13 @@ private String generateId(IdentityProvider identityProvider) {
         return identityProvider.getId();
     }
 
+    private String generateUniqueName(Bundle bundle, String name, IdentityProvider identityProvider) {
+        if (identityProvider != null && (identityProvider.getType() == BIND_ONLY_LDAP || identityProvider.getType() == FEDERATED)) {
```

Comment:
why do you want to apply the naming format conditionally?

Reply:
Fine, as this entity is not being tracked, we want to skip the unique naming. I'm FINE with it.

Reply:
we should apply the unique naming only for entities which are migrating through DevPlugin.
In this case ex: full LDAP policy author will add this entity to target by other tools, so I thought not to change it.